query/api/src/main/java/org/codice/ddf/admin/api/fields/EnumValue.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.api.fields;
+
+public interface EnumValue<T> {
```

Comment:
A `EnumValue` should never be used like the fields operate. For example, the `ObjectField` should never have `EnumValue` as it's inner field, it should use `EnumField` instead. I see the similarities but I don't think `Field` fits `EnumValue`